models/userDao.js

```diff
@@ -0,0 +1,71 @@
+const appDataSource = require('./appDataSource');
+const { CustomError } = require('../utils/error');
+
+const getUserByKakaoId = async (kakaoId) => {
+  try {
+    const [result] = await appDataSource.query(
+      `SELECT 
+      id,
+      kakao_id as kakaoId,
+      account_email as accountEmail,
+      profile_image as profileImage,
+      profile_nickname as profileNickname, 
+      birthday,
+      gender,
+      age_range as ageRange
+        FROM users WHERE kakao_id = ?`,
+      [kakaoId]
```

Comment:
```suggestion
      age_range as ageRange
        FROM users WHERE kakao_id = :kakao_id
      `, { kakao_id: kakaoId }
```

- sql문의 가독성을 위해서 위와 가튼 형태로도 변수기입이 가능하다는 점을 참고차 공유드립니다~

Reply:
- 물음표 ? place holer 사용이 많아 질 때 사용 가능할 수 있으니 추후 API 기획에서 한 번 사용을 해보세요!~